src/DotNetWorker.Core/DotNetWorker.Core.csproj

```diff
@@ -9,7 +9,8 @@
     <RootNamespace>Microsoft.Azure.Functions.Worker.Core</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <MinorProductVersion>10</MinorProductVersion>
-    <VersionSuffix>-preview1</VersionSuffix>
+    <PatchProductVersion>1</PatchProductVersion>
```

Comment:
Temporarily bumped patch version only for getting a package which works with apirevew diff screen. It does not work with the bump on preview version. P